src/views/About.tsx

```diff
@@ -6,107 +6,86 @@ import BulletImg from "assets/Across-logo-bullet.svg";
 const About: FC = () => {
   return (
     <Wrapper>
-      <Hero>
+      <HeroBg>
         <HeroHeader>
           Instantly Send Assets from L2 Rollups to L1 Ethereum
         </HeroHeader>
-      </Hero>
-      <BodyWrapper>
-        <BulletWrapper>
-          <BulletImageWrapper>
-            <BulletImage src={BulletImg} alt="across_logo" />
-          </BulletImageWrapper>
-          <BulletTextWrapper>
-            <BulletHeader>Fast Transfers</BulletHeader>
+      </HeroBg>
+      <Body>
+        <Bullet>
+          <Image src={BulletImg} alt="across_logo" />
+          <TextWrapper>
+            <BulletHeader>Instantaneous Liquidity</BulletHeader>
             <BulletText>
-              Lorem ipsum SEO sit amet, consectetur adipiscing elit. Sed vitae
-              tristique erat. Maecenas suscipit commodo quam, vitae scelerisque
-              elit vestibulum id.
-              <br /> <br />
-              Mauris sed magna tempor, feugiat elit placerat, sagittis urna.
-              Suspendisse porttitor neque ex, et rutrum here is a link sed. Cras
-              sodales nunc ac nunc.
+              Assets are bridged and available for use on mainnet almost
+              instantly.
             </BulletText>
-          </BulletTextWrapper>
-        </BulletWrapper>
-        <BulletWrapper>
-          <BulletImageWrapper>
-            <BulletImage src={BulletImg} alt="across_logo" />
-          </BulletImageWrapper>
-          <BulletTextWrapper>
-            <BulletHeader>Across different chains</BulletHeader>
+          </TextWrapper>
+        </Bullet>
+        <Bullet>
+          <Image src={BulletImg} alt="across_logo" />
+          <TextWrapper>
+            <BulletHeader>Secure</BulletHeader>
             <BulletText>
-              Cosectetur adipiscing SEO. Sed vitae tristique erat. Maecenas
-              suscipit commodo quam, vitae scelerisque elit vestibulum id.
-              <br /> <br />
-              Mauris sed magna tempor, feugiat elit placerat, sagittis urna.
+              Powered By UMA. Transfers are secured by UMA's Optimistic Oracle,
```

Comment:
Tom and I decided to swap this so it doesn't sound like it's powered by the UMA _token_. We mistakenly forgot to update the document.

```suggestion
              Powered By UMA protocol. Transfers are secured by UMA's Optimistic Oracle,
```